src/aframe-streetmix-parsers.js

```diff
@@ -788,15 +788,10 @@ module.exports.processSegments = processSegments;
 // test - for streetObject of street 44 and buildingElementId render 2 building sides
 function processBuildings (left, right, streetWidth, showGround, length) {
   const buildingElement = document.createElement('a-entity');
-  // const clonedObjectRadius = (length - 12) / 2;
   const clonedObjectRadius = 70;
```

Comment:
linting error `clonedObjectRadius` is no longer used in this function
either we can get rid of this or set to `0.45*length` and add this back to the end of lines 876 and 888

Reply:
https://github.com/3DStreet/3dstreet/pull/154/commits/722d2a958bf7621e6209a5e6cac449a7a99398c1